appcelerator/titanium_mobile
# fix(android): remote policy encryption error dialog handling

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28372

**Summary:**
- This is a regression introduced by PR #12509 that was caught before release.
- Fixes "remote" policy encryption issue where error dialog will not appear on startup due to lack of internet connection or if device was "rooted".
- Cause by removal of `verifyCustomModule()` method needed by "appc" CLI. See: [link](https://github.com/appcelerator/appc-cli-titanium/blob/master/hook/titanium.js#L609-L615)

**Test:**
1. Create a Titanium app project.
2. Add the following line to the "tiapp.xml" file.
`<property name="appc-sourcecode-encryption-policy">remote</property>`
3. Disable an Android device's internet connection.
4. Build and run the app to that Android device.
5. Verify a "Security Violation Detected" dialog appears on app startup.
6. Re-enable the device's internet connection.
7. Re-launch the app and verify it starts up without error.
